javascript/ql/src/experimental/poi/PoI.qll

```diff
@@ -196,15 +203,17 @@ private module StandardPoIs {
 import StandardPoIs
 
 /**
- * A tagging interface for the standard points of interest.
+ * A tagging interface for a custom point of interest that should be
+ * enabled in the absence of an explicit
+ * `PoIConfiguration::enabled/1`.
  */
-abstract private class StandardPoI extends PoI {
+abstract class DefaultEnabledPoI extends PoI {
```

Comment:
The name "default" looks a weird from the user's perspective. When you're specifically selecting which PoIs to use, it doesn't feel right to call them "defaults".

What if we called it `ActivePoI` instead?